services/QuillLMS/config/locales/en.yml

```diff
@@ -30,6 +30,7 @@ en:
     errors:
       activity_not_assigned: 'Sorry, you do not have access to this activity because it has not been assigned to you. Please contact your teacher.'
       activity_pack_not_assigned: 'Sorry, you do not have access to this activity pack because it has not been assigned to you. Please contact your teacher.'
+      activity_locked: 'Sorry, you do not have access to this activity as it is locked.  Please contact your teacher.'
```

Comment:
Do we want a more complex error here so that if a student does contact their teacher, and that teacher comes to look at their computer, the teacher can quickly figure out why the link isn't working?  Not sure exactly what that text would look like, but I think we definitely want to avoid students encountering this message, calling their teachers, and then their teachers reaching out to Support.